salt/tests/unit/modules/files/test_metalk8s_network.yaml

```diff
@@ -274,7 +273,7 @@ get_control_plane_ingress_external_ips:
       - 1.1.1.1
 
   # 3. Nominal multi node (using bootstrap IP)
-  - cp_ingress_ip_ret: 1.1.1.1
```

Comment:
No sorry I removed my comment here it was wrong you can keep the previous one since the test I mentioned is the one after